app/scripts/controllers/preferences.js

```diff
@@ -505,6 +534,19 @@ export default class PreferencesController {
     return this.store.getState().disabledRpcMethodPreferences;
   }
 
+  ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+  async prepareMmiPortfolio() {
+    if (!process.env.IN_TEST) {
+      this.handleMmiPortfolio().then((mmiDashboardData) => {
```

Comment:
Nit: `async/await` is preferred over `.then`/`.catch` because it produces better stack traces (with `.then` the stack would end here)

Reply:
You're right, I'll change it